packages/nexus-sdk/src/Archive/__tests__/index.spec.ts

```diff
@@ -36,6 +36,20 @@ describe('Archive', () => {
       expect(fetchMock.mock.calls[0][0]).toEqual(
         'http://api.url/v1/archives/org/project/archiveId?as=text',
       );
+      expect(fetchMock.mock.calls[0][1].headers).toEqual({
+        Accept: 'application/x-tar',
+      });
+      expect(fetchMock.mock.calls[0][1].method).toEqual('GET');
+    });
+
+    it('calls hhtpGet with the correct header', async () => {
+      await archive.get('org', 'project', 'archiveId', { as: 'json' });
```

Comment:
Do we need multiple test cases with different values for "as"? Also, in that case , they all cane be in a new describe block.